src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -55,8 +55,8 @@
 #include <openrct2/world/tile_element/SurfaceElement.h>
 #include <openrct2/world/tile_element/TrackElement.h>
 
-constexpr int8_t kDefaultSpeedIncrement = 2;
-constexpr int8_t kDefaultMinimumSpeed = 2;
+constexpr int8_t kDefaultSpeedIncrement = 1;
```

Comment:
What are people's thoughts on forcing valid speeds on rides with the legacy flag? I think we shouldn't. Invalid speeds are rounded down to the next valid legacy speed which isn't a major issue, and people should be using the new system anyway.